### PR TITLE
auto create users when they join #general

### PR DESCRIPTION
**Pull request by emmawilsonix**

### CODE REVIEW

File: api/app.py [hunk 1]
```diff
@@ -97,6 +100,57 @@ def try_grant_points(source_user_email, mentioned_user_email, points):
...
+        print("member joined a channel we don't care about...")
+        return
+
+    # Get user info and add them to the DB.
+    joined=message["user"]
+    joined_user = slack_client.users_info(user=joined)
+    joined_user_email = joined_user["user"]["profile"]["email"]
+    team = try_add_user(joined_user_email)
+    if team is None:
+        msg = """Hey <@{joined}> :wave: I'm <@{snitch}>! Looks like you joined #general but something went wrong so I wasn't able to assign you a team.
```
Comment (ouellettecameron) [inline api/app.py:119]: This seems to imply that we do actually care about the `#general` channel, whereas above we return early when the channel is `#general`

File: api/app.py [hunk 2]
```diff
@@ -97,6 +100,57 @@ def try_grant_points(source_user_email, mentioned_user_email, points):
...
+
+You can award points to other IXers by @ mentioning me and someone you want to give points to on slack! Just tell me how many points to give them and I am on it! 
+
+Looks like you've been sorted into *{team}*! Congratulations that's one of the best ones.
+
+⚡ Make sure to check out the leaderboard <https://snitch-leaderboard.herokuapp.com/|here>! ⚡
+
+⚡ And get to snitching! ⚡
+
+Much Love :heart: <@snitch>""".format(joined=joined, snitch=SLACKBOT_USERID, team="hufflepuff")
```
Comment (ouellettecameron) [inline api/app.py]: `team=team` instead of hufflepuff ?

File: api/app.py (hunk 1 above)
Comment (emmawilsonix) [reply api/app.py:119]: above we skip when we're *not* in general

File: api/app.py (hunk 2 above)
Comment (emmawilsonix) [reply api/app.py]: fixed

File: api/app.py (hunk 1 above)
Comment (ouellettecameron) [reply api/app.py:119]: Good point, coolio :)

**Pull request merged**